slotfills/sections/radio.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { PanelRow, RadioControl } from '@wordpress/components';
+import Field from '@/interfaces/field';
+
+interface RadioProps {
+  field: Field;
+  index?: number;
```

Comment:
```suggestion
  index?: number | null;
```
Suggesting only because index is being defaulted to `null`.